models/data.model.js

```diff
@@ -45,3 +45,33 @@ exports.fetchArticles = () => {
       }
     });
 };
+
+exports.fetchComments = (id, order_by) => {
+  let selectCommentsQueryStr = `SELECT * FROM comments`;
+  const queryParams = [];
+
+  if (id) {
+    selectCommentsQueryStr += ` WHERE article_id = $1`;
+    queryParams.push(id);
+  }
```

Comment:
Again, optional, but im sure theres always an id, so probably dont need the splitting up here